tests/src/testmaterial.cpp

```diff
@@ -305,6 +303,18 @@ int main(int argc, char** argv) {
   auto nhRed8 = makeVanishingStrain<Impl::MatrixIndexPair{1, 1}, Impl::MatrixIndexPair{2, 2}>(nh);
   t.subTest(testMaterial(nhRed8));
 
+  auto muesliLin = Materials::Muesli::makeLinearElasticity(matPar);
+  t.subTest(testMaterial(muesliLin));
+
+  auto muesliSVK = Materials::Muesli::makeSVK(matPar);
+  t.subTest(testMaterial(muesliSVK));
+
+  auto muesliNeoHooke = Materials::Muesli::makeNeoHooke(matPar, false);
+  t.subTest(testMaterial(muesliNeoHooke));
+
+  auto muesliNeoHookeReg = Materials::Muesli::makeNeoHooke(matPar, true);
```

Comment:
May be we can add similar tests for other finite strain material models?

Reply:
Ah yes I wanted to do this, I was just unsure about good values for the material parameters